spec/features/card_management_spec.rb

```diff
@@ -13,19 +14,21 @@
       visit new_card_path
       fill_in "card_original_text", with: "way"
       fill_in "card_translated_text", with: "путь"
+      select "English", from: 'Deck'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.